src/thermostate/plotting.py

```diff
@@ -58,10 +57,15 @@ def plot(self, x_axis, y_axis):  # pragma: no cover
         """Hold the place of a plot function that a child class must establish."""
         pass
 
-    def add_state(self, state, key=None):
+    def add_state(self, state, key=None, label = None):
```

Comment:
Typical Python style is to not add spaces around `=` in function calls and definitions, so this should be:

```python
    def add_state(self, state, key=None, label=None):
```